src/Main/Main.js

```diff
@@ -282,18 +286,41 @@ class Main extends Component {
   };
 
   render() {
-    const { onSelectRow, disableRecordCreation, onComponentWillUnmount, showSingleResult, browseOnly } = this.props;
+    const {
+      browseOnly,
+      disableRecordCreation,
+      mutator,
+      onComponentWillUnmount,
+      onSelectRow,
+      resources,
+      showSingleResult,
+      stripes,
+      visibleColumns,
+    } = this.props;
     const resultsFormatter = {
-      'Name': data => _.get(data, ['name'], ''),
-      'Code': data => _.get(data, ['code'], ''),
-      'Description': data => _.get(data, ['description'], ''),
-      'Status': data => _.toString(_.get(data, ['status'], '')),
+      'Name': data => get(data, ['name'], ''),
+      'Code': data => get(data, ['code'], ''),
+      'Description': data => get(data, ['description'], ''),
+      'Status': data => toString(get(data, ['status'], '')),
+      'isVendor': data => {
```

Comment:
I think it could be written shorter and easier to read:
```
'isVendor': ({ isVendor }) => <FormattedMessage id={`ui-organizations.main.isVendor.${isVendor ? 'yes' : 'no'}`} />,
```

Reply:
Cool. Thank you